### PR TITLE
Add CP Sapling + CP Sapling w/ spi flash boards

### PR DESCRIPTION
**Pull request by skerr92**
Hey there!

I'm adding two variants of boards that my company has been working on to the CircuitPython.py file for Mu. Hopefully there isn't any issues. I'll commit any changes necessary if any issues arise.

Thanks!

### CODE REVIEW

**Comment by dybber** [comment]
Thanks for the addition! Could you provide a link here in a comment, documenting that these boards have these VID/PIDs?

**Comment by skerr92** [comment]
Here is the PID for CP Sapling M0 board: <a href="https://github.com/pidcodes/pidcodes.github.com/tree/master/1209/4DDD">CP Sapling</a> 
The PID request for the CP Sapling M0 w/ SPI Flash is PR'd and waiting for final approval from pidcodes: <a href="https://github.com/pidcodes/pidcodes.github.com/pull/586">PR for CP Sapling w/ SPI Flash</a>

**Comment by carlosperate** [comment]
The other PID was approved in https://github.com/pidcodes/pidcodes.github.com/pull/560

Entries can be seen in https://pid.codes/pids/:

image

**Review by carlosperate — APPROVED** [review]

**Pull request merged**
